contracts/mainnet/connectors/sushi-incentive/main.sol

```diff
@@ -34,17 +35,19 @@ abstract contract SushipswapIncentiveResolver is Helpers, Events {
         token1 = changeEthAddrToWethAddr(token1);
         token2 = changeEthAddrToWethAddr(token2);
         amount = getUint(getId, amount);
-        (uint256 _pid, uint256 _version, address lpTokenAddr) = _getPoolId(
-            token1,
-            token2
-        );
-        setUint(setId, _pid);
-        require(_pid != uint256(-1), "pool-does-not-exist");
-        TokenInterface lpToken = TokenInterface(lpTokenAddr);
+        if(data.poolId == uint256(-1)|| data.version <= 0 || data.lpToken == address(0)){
```

Comment:
```suggestion
        if(data.poolId == 0 || data.version == 0 || data.lpToken == address(0)){
```

I think this would be more appropriate?

Reply:
Hey there exists a pool with poolId == 0

Reply:
Ahh! okay. Let's use `uint(-1)` only